packages/terra-table/src/subcomponents/Section.jsx

```diff
@@ -95,6 +95,33 @@ const propTypes = {
    * Function that is called when a collapsible section is selected. Parameters: `onSectionSelect(sectionId)`
    */
   onSectionSelect: PropTypes.func,
+
+  /**
+   * @private
+   * A Unique Identifier of the [column](/components/cerner-terra-framework-docs/table/about#column).
+   * If provided, column with specified identifier will be highlighted in data-grid.
+   *
+   * ![IMPORTANT](https://badgen.net/badge/UX/Design-Standards/blue) The column highlight feature should be limited specifically to
+   * time and timeline concepts only, best used with special instruction and guidance from User Experience to ensure proper standards.
+   */
+  columnHighlightId: PropTypes.string,
+
+  /**
+   * @private
+   * Id of the first row in table
+   */
+  firstRowId: PropTypes.string,
+
+  /**
+   * @private
+   * Id of the last row in table
+   */
+  lastRowId: PropTypes.string,
+
+  /**
+   * The color to be used for highlighting a column.
+   */
+  colorHighlightColor: PropTypes.oneOf(['orange', 'green']),
```

Comment:
This should also be a private prop

Reply:
https://github.com/cerner/terra-framework/pull/1985/commits/f316f91d6aa32289ef93a1a41ef44f72e62761e8#diff-64d05a135418b5eaa187214d1d0a5ed8df59d36a646018d75c2cf345b93a30e8